From 0ad871f12f7c1cc4e6e04a13d236c3e8abd2e2d6 Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Fri, 20 Dec 2024 11:27:20 +0800
Subject: [PATCH 347/652] WIN2030-14758:feat:Select cipher as default

Changelogs:
1.Select CONFIG_CRYPTO_DEV_ESWIN_EIC770x in eic7702_defconfig
2.Enable d1_ipc_scpu in eic7702-evb-a1.dts and eic7702-evb-a1-interleave.dts

Signed-off-by: linmin <linmin@eswincomputing.com>
Change-Id: I9c2654a07c12727c6321a19f278a68dafcbf0a9d
---
 arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts | 2 +-
 arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts            | 2 +-
 arch/riscv/configs/eic7702_defconfig                    | 2 ++
 3 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts b/arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts
index 1470d3990cd5..b7ea2eea76cb 100644
--- a/arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-evb-a1-interleave.dts
@@ -395,7 +395,7 @@ &d0_ipc_scpu {
 	status = "okay";
 };
 &d1_ipc_scpu {
-	status = "disabled";
+	status = "okay";
 };
 
 &d0_lpcpu {
diff --git a/arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts b/arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts
index 058a28049e0f..12dbdb1c7418 100644
--- a/arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-evb-a1.dts
@@ -373,7 +373,7 @@ &d0_ipc_scpu {
 	status = "okay";
 };
 &d1_ipc_scpu {
-	status = "disabled";
+	status = "okay";
 };
 
 &d0_lpcpu {
diff --git a/arch/riscv/configs/eic7702_defconfig b/arch/riscv/configs/eic7702_defconfig
index 10811521fe52..e2cb568415e9 100644
--- a/arch/riscv/configs/eic7702_defconfig
+++ b/arch/riscv/configs/eic7702_defconfig
@@ -9,6 +9,7 @@ CONFIG_PREEMPT_VOLUNTARY=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_NUMA_BALANCING=y
+# CONFIG_NUMA_BALANCING_DEFAULT_ENABLED is not set
 CONFIG_CGROUPS=y
 CONFIG_MEMCG=y
 CONFIG_BLK_CGROUP=y
@@ -807,6 +808,7 @@ CONFIG_CRYPTO_HMAC=m
 CONFIG_CRYPTO_MD5=y
 CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_DEV_VIRTIO=y
+CONFIG_CRYPTO_DEV_ESWIN_EIC770x=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
 CONFIG_DMA_CMA=y
-- 
2.53.0

